### PR TITLE
Fixed some spelling

### PR DESCRIPTION
**Pull request by danperry**
please merge this guys, its important

### CODE REVIEW

**Comment by bcye** [comment]
Great awesome

**Comment by bcye** [comment]
Im conversing

File: PIE/group_edit.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ection 1
```
Comment (danperry) [inline PIE/group_edit.txt]: You missed fixing this
Should be:
```
Section 1
```

Heres some more code: `more code`

Comment (bcye) [reply PIE/group_edit.txt]: Awesome

**Review by bcye — APPROVED** [review]
Awesome work

**Pull request merged**